modules/administration/pages/security.adoc

```diff
@@ -164,3 +164,30 @@ $> rpm --import http://suma-refhead-srv.mgr.suse.de/pub/company.key
 ====
 For salt managed systems it might make sense to use a state to trust GPG keys.
 ====
+
+
+== Mirror Source Packages
+
+If you build your own packages locally, or if you require the source code for your packages for legal reasons, it is possible to mirror the source packages on your local system.
+
+[NOTE]
+====
+Note that this can consume a significant amount of disk space.
+====
+
+.Procedure: Mirroring Source Packages
+. Open the [filename]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+----
+server.sync_source_packages = 1
+----
+. Restart the spacewalk service to pick up the changes:
+----
+spacewalk-service restart
+----
+
+Currently, this feature can only be enabled globally for all repositories.
+It is not possible to select individual repositories for mirroring.
+
+Once this feature has been activated, the source packages will become available in the {productname} {webui}.
+They will be shown as sources for the binary package, and can be downloaded directly from the {webui}.
```

Comment:
I need to check, but maybe we have also an API endpoint to get the package.

Reply:
We can do another patch once you have that info.